File: server/Models/Keep.cs
namespace keepr.Models;

public class Keep
{
    public string Name { get; set; }
    public string Description { get; set; }
    public string Img { get; set; }
    public int Views { get; set; }
    public string CreatorId { get; set; }
    public Profile Creator { get; set; }
    public int Kept { get; set; }
}

File: client/src/models/Keep.js
import { DatabaseItem } from "./DatabaseItem.js";
import { Profile } from "./Profile.js";


export class Keep extends DatabaseItem {
    constructor(data){
        super(data)
        this.name = data.name;
        this.description = data.description;
        this.img = data.img;
        this.views = data.views;
        this.creatorId = data.creatorId;
        this.creator = new Profile(data.creator);
        this.kept = data.kept;
    }
}